extensions/azure/cosmos/assetindex-cosmos/build.gradle.kts

```diff
@@ -17,13 +17,18 @@ plugins {
 }
 
 val cosmosSdkVersion: String by project
+val okHttpVersion: String by project
+val jodahFailsafeVersion: String by project
 
 dependencies {
     api(project(":spi"))
     api(project(":common:util"))
     api(project(":extensions:azure:cosmos:cosmos-common"))
     api(project(":extensions:dataloading"))
+
     implementation("com.azure:azure-cosmos:${cosmosSdkVersion}")
+    implementation("com.squareup.okhttp3:okhttp:${okHttpVersion}")
```

Comment:
didn't find a usage for this, so maybe it can go away?

Reply:
I didn't look too much at third-party libraries in this PR. I'll remove this one though.